README.md

```diff
@@ -18,6 +18,7 @@ require 'craigslister'
 client = Craigslister.new(
   item: 'Couch',
   area: 'austin', # optional (defaults to 'sfbay')
+  section: 'motorcycles', # optional (defaults to 'sss')
   low:  500,      # optional
   high: 2000,     # optional
 )
```

Comment:
@pgdaniel The only minor issue that I can see is the sections in the url (on Craigslist) are abbreviated, so the 'motorcycles' section becomes 'mca' in the url (e.g., `https://sfbay.craigslist.org/search/mca?query=harley%20davidson`). Should be expect a user to input the abbreviated section, or should we allow them to input something like 'motorcycles' and do the conversion for them? What do you think?